src/components/authorize/claims-config.ts

```diff
@@ -0,0 +1,38 @@
+export const expectedClientId = "orchestrationAuth";
```

Comment:
For adherence to naming conventions, I would capitalize the immutable variable. Consider using `EXPECTED_CLIENT_ID` or `CLIENT_ID`.